src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Overview/ThorchainSaversOverview.tsx

```diff
@@ -90,15 +90,18 @@ export const ThorchainSaversOverview: React.FC<OverviewProps> = ({
   const assets = useAppSelector(selectAssets)
   const asset = useAppSelector(state => selectAssetById(state, assetId))
 
+  const isRunePool = assetId === thorchainAssetId
+
   const { isLoading: isMockDepositQuoteLoading, error } = useGetThorchainSaversDepositQuoteQuery({
     asset,
     amountCryptoBaseUnit: BigNumber.max(
       THORCHAIN_SAVERS_DUST_THRESHOLDS_CRYPTO_BASE_UNIT[assetId],
       toBaseUnit(1, asset?.precision ?? 0),
     ),
-    enabled: assetId !== thorchainAssetId,
+    enabled: !isRunePool,
   })
 
+  // @TODO: Find how we can handle this for RUNEPool
```

Comment:
Remove me? Looks like this will be automagically handled for RUNEPool since this expression should ever evaluate to `true`

https://github.com/shapeshift/web/blob/d11393b2e5a10eacf84876bdf4b5c7ccf2a24665/src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Overview/ThorchainSaversOverview.tsx#L340-L359

Reply:
Yeah you're right, some wip dust I guess, thanks for the catch, fixed in [6b759a9](https://github.com/shapeshift/web/pull/7440/commits/6b759a95e9c9dfc87aedcf9f26d821336ca28f44)